web/index.php

```diff
@@ -13,9 +13,126 @@
 //TODO: manage json export
 //TODO: manage type from DDB
 //TODO: manage type from Flarm
-//TODO: manage timezone (query + display)
 //date_default_timezone_set("Zulu");
 
+if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+	if (isset($_POST['tf']))
+		$_GET['tf'] = $_POST['tf'];
+	if (isset($_POST['d']))
+		$_GET['d'] = $_POST['d'];
+}
+
+class timeFormatHelper{
+	//TODO correct handling for flights over midnight
+	public int $timestamp = 0;
```

Comment:
We need to remove the "int" to be compatible with PHP 7.3 currently powering flightlog.glidernet.org